cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -837,7 +841,9 @@ public MapAndStatus(T loadMap, K status) {
                 return requestRebalance(reconciliation, host, apiClient, kafkaRebalance, true, rebalanceOptionsBuilder);
             default:
                 LOGGER.warnCr(reconciliation, "Ignore annotation {}={}", ANNO_STRIMZI_IO_REBALANCE, rebalanceAnnotation);
-                return Future.succeededFuture(new MapAndStatus<>(null, buildRebalanceStatusFromPreviousStatus(kafkaRebalance.getStatus(), validate(reconciliation, kafkaRebalance))));
+                Set<Condition> conditions = validate(reconciliation, kafkaRebalance);
+                conditions.add(StatusUtils.buildWarningCondition("Unsupported/Unknown annotation value", "The rebalance resource supports the following annotations : approve, refresh, stop."));
```

Comment:
Is it possible to add the actual annotation text the user has attached to the warning message?

Reply:
I am not sure since the annotation is converted to unknown by this step. But I can try that for sure

Reply:
I think Tom meant something like 
`conditions.add(StatusUtils.buildWarningCondition("Unsupported/Unknown annotation value" + rebalanceAnnotation, "The rebalance resource supports the following annotations : approve, refresh, stop."));`

Reply:
@sknot-rh Actually the  annotation is changed to `unknown`. For eg If we pass `Dave` as an annotation then it will get converted to unknown and then this if condition gets executed.

Reply:
I think maybe Tom meant that we should have that wrong annotation in the message which is `Dave` in the above eg

Reply:
You could add a field to the Rebalance Annotation enum to store the supplied annotation text? Just an idea.